components/Comment/CommentEditor.tsx

```diff
@@ -147,7 +147,9 @@ const CommentEditor = ({
       if (_isEmpty) {
         forceShowPlaceholder({
           quillRef,
-          placeholderText: placeholder,
+          placeholderText:
+            commentTypes.find((ctype) => ctype.value === commentType)
+              ?.placeholder || placeholder,
```

Comment:
The type-specific placeholder weren't showing because they were getting overwritten by the default `placeholder`. This fixes that issue.